src/controller/python/chip/ChipStack.py

```diff
@@ -164,6 +165,31 @@ def Wait(self, timeoutMs: int = None):
             return self._res
 
 
+class AsyncSimpleCallableHandle:
```

Comment:
Because of the class description, maybe just call this `AsyncioCallableHandle`, and just populate `_loop` and `_future` yourself and not in `CallAsync`